http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/JsonPayloadIT.java

```diff
@@ -78,67 +81,64 @@ public void sendLargeComplexJSONPayload() throws IOException {
                 .get("/persons")
                 .then()
                 .statusCode(OK_STATUS_CODE)
-                .body(containsString("Hello, Singleton Middleton! You have 4 unread messages."));
+                .body(containsString("Hello, Singleton Middleton! You have 4 unread messages.")).extract().response();
 
     }
 
     @Test
-    public void sendJSONWithEmptyFieldsPayload() throws IOException {
+    public void sendJSONWithEmptyFieldsPayloadNoResourceValidation() throws IOException {
         String jsonPayload = readJSONFile("football-empty-teams.json");
-
-        getApp().given()
+        Response response = getApp().given()
                 .contentType(ContentType.JSON)
                 .body(jsonPayload)
                 .when()
                 .post("/football/upload-football-json")
                 .then()
-                .statusCode(BAD_REQUEST_STATUS_CODE)
-                .body(containsString("Some fields are empty or null"));
+                .statusCode(CREATION_STATUS_CODE).extract().response();
+        JsonPath jsonPath = response.jsonPath();
+        // Check that there are 2 teams with empty colorTShirt fields
+        assertThat(jsonPath.getList("teams.findAll { it.colorTShirt == '' }"), hasSize(2));
     }
 
     @Test
     public void sendInvalidJSONPayload() throws IOException {
         String jsonPayload = readJSONFile("football-teams_invalid.json");
-
         getApp().given()
                 .contentType(ContentType.JSON)
                 .body(jsonPayload)
                 .when()
                 .post("/football/upload-football-json")
                 .then()
-                .statusCode(BAD_REQUEST_STATUS_CODE)
-                .body(containsString(
-                        "Invalid JSON - JsonProcessingException : Unexpected character ('{' (code 123)): was expecting comma to separate Array entries"));
+                .statusCode(INTERNAL_SERVER_ERROR_STATUS_CODE)
+                .body(containsString("Internal error: Invalid token=CURLYOPEN"))
```

Comment:
```suggestion
                .body(containsString("Internal error: Invalid token=CURLYOPEN"),
                       containsString("Expected tokens are: [COMMA]"));
```

It should be possible to check it like suggestion above